cli/cmd/resource_run_cmd.go

```diff
@@ -5,75 +5,48 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/kubeshop/tracetest/cli/cmdutil"
 	"github.com/kubeshop/tracetest/cli/config"
 	"github.com/kubeshop/tracetest/cli/openapi"
-	"github.com/kubeshop/tracetest/cli/pkg/fileutil"
 	"github.com/kubeshop/tracetest/cli/runner"
 	"github.com/spf13/cobra"
+
+	cloudCmd "github.com/kubeshop/tracetest/cli/cloud/cmd"
 )
 
 var (
-	runParams = &runParameters{}
+	runParams = &cmdutil.RunParameters{}
 	runCmd    *cobra.Command
 )
 
 func init() {
 	runCmd = &cobra.Command{
 		GroupID: cmdGroupResources.ID,
-		Use:     "run " + runnableResourceList(),
-		Short:   "run resources",
-		Long:    "run resources",
+		Use:     fmt.Sprintf("run [%s]", runnableResourceList()),
+		Short:   "Run tests and test suites",
+		Long:    "Run tests and test suites",
 		PreRun:  setupCommand(WithOptionalResourceName()),
 		Run: WithResourceMiddleware(func(ctx context.Context, _ *cobra.Command, args []string) (string, error) {
-			resourceType, err := getResourceType(runParams, resourceParams)
-			if err != nil {
-				return "", err
-			}
+			if cliConfig.Jwt != "" {
+				exitCode, err := cloudCmd.RunMultipleFiles(ctx, runParams, &cliConfig, runnerRegistry, output)
 
-			r, err := runnerRegistry.Get(resourceType)
-			if err != nil {
-				return "", fmt.Errorf("resource type '%s' cannot be run", resourceType)
-			}
-
-			orchestrator := runner.Orchestrator(
-				cliLogger,
-				config.GetAPIClient(cliConfig),
-				variableSetClient,
-			)
-
-			if runParams.EnvID != "" {
-				runParams.VarsID = runParams.EnvID
-			}
-
-			runParams := runner.RunOptions{
-				ID:              runParams.ID,
-				DefinitionFile:  runParams.DefinitionFile,
-				VarsID:          runParams.VarsID,
-				SkipResultWait:  runParams.SkipResultWait,
-				JUnitOuptutFile: runParams.JUnitOuptutFile,
-				RequiredGates:   runParams.RequriedGates,
-			}
-
-			exitCode, err := orchestrator.Run(ctx, r, runParams, output)
-			if err != nil {
+				ExitCLI(exitCode)
 				return "", err
 			}
 
-			ExitCLI(exitCode)
-
-			// ExitCLI will exit the process, so this return is just to satisfy the compiler
-			return "", nil
+			return runSingleFile(ctx)
 
 		}, runParams),
 		PostRun: teardownCommand,
 	}
 
-	runCmd.Flags().StringVarP(&runParams.DefinitionFile, "file", "f", "", "path to the definition file")
-	runCmd.Flags().StringVar(&runParams.ID, "id", "", "id of the resource to run")
+	runCmd.Flags().StringSliceVarP(&runParams.DefinitionFiles, "file", "f", []string{}, "path to the definition file (can be defined multiple times)")
+	runCmd.Flags().StringVarP(&runParams.ID, "id", "", "", "id of the resource to run (can be defined multiple times)")
```

Comment:
I thought about having multiple IDs, but I reverted the changes and forgot to change here:

```suggestion
	runCmd.Flags().StringVarP(&runParams.ID, "id", "", "", "id of the resource to run")
```

Reply:
If you don't mind, I'll fix this in the next PR, I want to have this merged 🥴